### ISSUE TITLE
hyperparams for CIFAR100 dataset

### ISSUE THREAD

**Issue by lileiwu** [liuhao-lh/SMD#2]
Thanks for the great work! I tried to reproduce the results in table 1 and can't get the reported results. So I want to check whether the hyperparams is incorrectly used .
1. I used 2 rtx titan gpus and the hyperparams in paper and I noticed that the running scripts in README.md is on an 4-gpu machine, so you run the cifar-100 dataset on a 4-gpu machine with the reported hyperparams？
2. The common linear eval recipe is : a. lr : 30 with step decay [60, [80]] [CMC](https://arxiv.org/abs/1906.05849)  or  b. lr: 0.1 with cosine decay [MoCo v2](https://arxiv.org/abs/2003.04297), but I noticed you set learning rate as 30 with cosine decay, is it a typo?
3. The result in table 1 is confusing. The teacher for column 1 and column 3 is resnet32x4. However the knn and linear acc is different. While the teacher for column 1 and column 4 is different, but the results is the same. Is it a typo too?
